mmhuman3d/data/data_structures/human_data.py

```diff
@@ -915,6 +915,56 @@ def __check_value_len__(self, key: Any, val: Any) -> bool:
                 msg=err_msg, logger=self.__class__.logger, level=logging.ERROR)
         return ret_bool
 
+    def generate_mask_from_confidence(self, keys=None) -> None:
+        """Generate mask from keypoints' confidence. Keypoints that have zero
+        confidence in all occurrences will have a zero mask.Note that the last
```

Comment:
Missing space in ` a zero mask.Note that`.

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/142/commits/5d02720a0288b87459bbd0de6800a847c1af5625.